lib/atat-pipeline-stack.ts

```diff
@@ -0,0 +1,47 @@
+import * as cdk from "aws-cdk-lib";
+import * as pipelines from "aws-cdk-lib/pipelines";
+import * as ssm from "aws-cdk-lib/aws-ssm";
+import { Construct } from "constructs";
+import { AtatWebApiStack } from "./atat-web-api-stack";
+import { AtatIamStack } from "./atat-iam-stack";
+import { AtatNetStack } from "./atat-net-stack";
+
+export interface AtatProps {
+  environmentName: string;
+  vpcCidr?: string;
+}
+
+export interface AtatPipelineStackProps extends cdk.StackProps, AtatProps {
+  branch: string;
+  repository: string;
+  githubPatName: string;
+}
+
+export class AtatApplication extends cdk.Stage {
+  constructor(scope: Construct, id: string, props: cdk.StageProps & AtatProps) {
+    super(scope, id, props);
+    const iam = new AtatIamStack(this, `${props.environmentName}Iam`);
+    const net = new AtatNetStack(this, `${props.environmentName}Net`, { vpcCidr: props.vpcCidr });
+    const atat = new AtatWebApiStack(this, "Atat", {
+      environmentName: props.environmentName,
+      network: net,
```

Comment:
Does a `net` stack still get created if `props.vpcCidr` is `undefined`? It looks like a `vpcCidr` is not being supplied for the `app`, but I guess a default value might be provided by AWS. 

Also, do we still plan to have a shared vpc in sandbox, since we were having issues previously with running out of vpcs?

Reply:
It _should_ it will just use the default CIDR range. But good catch! I'll make sure we have a way to propagate that down to the net stack. The App stack shouldn't need it.

Reply:
Also... as far as VPC sharing, I was almost thinking that we just wouldn't care about the VPC in Sandbox environments? And make it a dev environment problem? And just rely on the "default" networking capabilities of the various services/resources in sandbox? Maybe that's a little too ambitious.

Because I _do_ still plan to deploy a `SandboxPipeline` (so that it can manage IAM), there actually _would_ be a Sandbox VPC (and IAM and API) that would track `develop` but could be configured to track a feature branch if that'd be useful for a ticket or for testing.

Reply:
Yeah, I don't think we care too much about the VPC in sandbox. Plus, we are doing less in AWS, so maybe things will be better. The VPCs were just resource intensive before

Cool, that makes sense (our IAM is every evolving). It might be nice to have that flexibility to track a feature branch for testing purposes, but I'll leave that up to you.